update.sh

```diff
@@ -154,25 +155,45 @@ print_basepackages_alpine() {
 	    dpkg \
 	    gnupg \
 	    wget \
-	    curl \
 	    bash \
 	    shadow \
 	    openjdk8 \
 	    zip \
-	    su-exec && \
+	    su-exec
+
+EOI
+}
+
+# Print cleanup for debian
+print_cleanup() {
+	cat >> $1 <<-'EOI'
+	# Reduce image size by removing files that are used only for building the image
+	RUN DEBIAN_FRONTEND=noninteractive apt-get remove -y dirmngr gnupg && \
+	    DEBIAN_FRONTEND=noninteractive apt-get autoremove -y && \
+	    rm -rf /var/lib/apt/lists/*
+
+EOI
+}
+
+# Print cleanup for alpine
+print_cleanup_alpine() {
+	cat >> $1 <<-'EOI'
+	# Reduce image size by removing files that are used only for building the image
+	RUN apk del dpkg gnupg && \
```

Comment:
Lgtm, the only improvement could be to use no-cache and to delete them als a group inside Alpine see also in the article which i referred to yesterday, but that is neccairly part of this PR

Reply:
I'll apply the alpine improvements but it seems there is already an index in the alpine image we build upon, so it to get rid of it the rm command still makes sense:

```
$ docker run --rm multiarch/alpine:amd64-v3.7 ls /var/cache/apk
APKINDEX.70c88391.tar.gz
```

Do you think we should add back in `procps` on debian for ps and top commands? It looks like the `udev` package is not installed by default (which would also install procps), but libudev we use is part of the image.

Reply:
The container process should also show up in top or ps on the host of course. :slightly_smiling_face: